src/main/java/uk/ac/cam/cl/dtg/util/ReflectionsUtil.java

```diff
@@ -0,0 +1,53 @@
+package uk.ac.cam.cl.dtg.util;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Enumeration;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class ReflectionsUtil {
+  public static List<Class<?>> getClasses(String packageName) throws ClassNotFoundException, IOException,
+      URISyntaxException {
+    ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
```

Comment:
Small one, but you could move this block into a private function that returns a list of directories, since that's what the next few lines are all about. That would improve readability.